frontend/src/scenes/insights/InsightTabs/RetentionTab/ReferencePicker.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Select } from 'antd'
+import { PercentageOutlined } from '@ant-design/icons'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { useActions, useValues } from 'kea'
+import { retentionTableLogic } from 'scenes/retention/retentionTableLogic'
+
+const options = {
+    total: {
+        value: 'total',
+        icon: <PercentageOutlined />,
+        label: 'Overall cohort',
+        description: 'Display retention values relative to initial cohort size',
+    },
+    previous: {
+        value: 'previous',
+        icon: <PercentageOutlined />,
+        label: 'Relative to previous period',
+        description: `
+            Display retention values relative to previous retention period. When
+            displayed as a line graph, this is what is sometimes called a J-Curve or
+            smile graph, and is intended to identify how quickly the dropoff of
+            users is tending towards zero
+        `,
+    },
+} as const
```

Comment:
The `as const` cast looks off. What are you trying to do here?

Reply:
I'm instructing that the dictionary is constant, so we should get narrow types for, e.g. string literals.

Reply:
(that was more important in a previous iteration when I was referencing values, not so much now)